ts/utils/hooks/bottomSheet.tsx

```diff
@@ -215,14 +215,7 @@ export const useIOBottomSheetAutoresizableModal = (
   );
 
   return useIOBottomSheetModal({
-    component: (
-      <View
-        style={{ paddingBottom: insets.bottom + bottomPadding }}
-        onLayout={handleContentOnLayout}
-      >
-        {component}
-      </View>
-    ),
+    component: <View onLayout={handleContentOnLayout}>{component}</View>,
```

Comment:
If you delete the padding bottom style for the View that wraps the content body of the autoresizable bottom sheet, for the bottom sheets without a footer you will have a 0 padding from the bottom as showed in the following screen:
<img src="https://github.com/pagopa/io-app/assets/34343582/e659ab07-7807-48fa-82a0-46274b64f5a9" width="250px" />

What do you think if the padding bottom styles are omitted only if there is a footer for that bottom sheet?

Reply:
we may then leave only the bottom value for the safearea inset, it should be enough

Reply:
That sounds good!

Reply:
added in [c8ebfb6](https://github.com/pagopa/io-app/pull/4691/commits/c8ebfb61ef3476802da6f3203207e9a3d0f1a030)